frontend/src/pages/manufacturing/Manufacturing.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Checkbox
+} from '@material-ui/core';
+import { Orders } from '../../interfaces/Orders';
+import { API_MAN } from '../../utils/api';
+import { Card } from '../../components';
+import './Manufacturing.scss';
+import OrderRow from './order-row/OrderRow';
+
+export default function Manufacturing() {
+  const [orders, setOrders] = useState<Orders[]>([]);
+
+  // Get orders from backend
+  const getOrders = async () => {
+    const request = await fetch(API_MAN, {
+      headers: { Authorization: `bearer ${localStorage.token}` }
```

Comment:
```suggestion
      headers: { Authorization: auth.getAuthorization() }
```
Now that we have auth, we better use it